bayesian/utils.py

```diff
@@ -64,17 +65,34 @@ def server_run_flow(flow_name, flow_args):
     :param flow_args: arguments for the flow
     :return: dispatcher ID handling flow
     """
-    current_app.logger.debug('Running flow {}'.format(flow_name))
```

Comment:
@yzainee are you facing any issues in `current_app.logger`?

Reply:
test cases do not work with current_app